steps/reorderHelperFiles/include/imagingtable.h

```diff
@@ -0,0 +1,171 @@
+#ifndef WSCLEAN_IMAGING_TABLE_H
+#define WSCLEAN_IMAGING_TABLE_H
+
+#include "imagingtableentry.h"
+
+#include <functional>
+#include <memory>
+#include <vector>
+
+#include "radlerHeaders/work_table.h"
+
+/**
+ * The ImagingTable contains ImagingTableEntry's and supports creating subtables
+ * with entries that have the same attribute.
+ * It supports the following grouping types:
+ * - IndependentGroup : Entries have an equal joinedGroupIndex in each group.
+ * - FacetGroup : Entries have an equal facetGroupIndex in each group.
+ *   Each group thus contains all entries that form a single image.
+ * - Facet : Entries have an equal facetIndex in each group.
+ *   Each group thus contains all entries for a single facet.
+ * - SquaredGroup : Entries have an equal squaredDeconvolutionIndex in each
+ *   group.
+ */
+class ImagingTable {
```

Comment:
Intentation should be fixed here too. Maybe apply clang-format to lint all the code